content/guides/10-minute-tutorial.md

```diff
@@ -901,7 +901,7 @@ function isItFriday(today) {
 
 {{% block "ruby" %}}
 ```ruby
-def is_it_friday(day)
+def is_it_friday?(day)
```

Comment:
Instead of ending the method with a question mark, would you please make it consistent the other way around (i.e. none of the methods having a question mark)?